src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -35,10 +35,10 @@
 
         <one-to-one field="defaultMeta" target-entity="Sulu\Bundle\MediaBundle\Entity\FileVersionMeta">
             <cascade>
-                <cascade-all />
+                <cascade-persist />
             </cascade>
 
-            <join-column name="idFileVersionsMetaDefault" referenced-column-name="id" nullable="true" on-delete="CASCADE" />
+            <join-column name="idFileVersionsMetaDefault" referenced-column-name="id" nullable="true" on-delete="SET NULL" /><!-- why this should on-delete="RESTRICT" to avoid remove defaultMeta because of the nature of databases we need to use here SET NULL else the FileVersion is never deleteable -->
```

Comment:
@Prokyonn currently we can use not `RESTRICT` here because of nature how the deletion inside the database work it currently always fails to remove if we restrict it. Best solution for future would be not using relations for `default...` connection instead we use a `defaultLocale` and getDefaultMeta returns via doctrine critiria the correct meta.